rss-core/rss-dao/src/test/java/es/upm/fiware/rss/dao/impl/test/SharingReportDaoImplTest.java

```diff
@@ -0,0 +1,119 @@
+package es.upm.fiware.rss.dao.impl.test;
```

Comment:
Include the header with the license and the copyright (Only UPM if you have created the file)